frontend/app/src/utils/calculation.ts

```diff
@@ -1,9 +1,16 @@
 import { Balance, BigNumber } from '@rotki/common';
+import { get } from '@vueuse/core';
+import { useIgnoredAssetsStore } from '@/store/assets';
 import { Zero } from '@/utils/bignumbers';
 
 export function assetSum(balances: { [asset: string]: Balance }) {
-  return Object.values(balances).reduce((sum, balance) => {
-    return sum.plus(balance.usdValue);
+  const { isAssetIgnored } = useIgnoredAssetsStore();
+
+  return Object.entries(balances).reduce((sum, [asset, balance]) => {
+    if (!isAssetIgnored || !get(isAssetIgnored(asset))) {
```

Comment:
Is there some condition can the `isAssetIgnored` be falsy?

Reply:
my bad, forgot to change it.